testing/mysql/integration_test.go

```diff
@@ -150,6 +150,62 @@ func TestIntegration_MYSQL_SimpleUse(t *testing.T) {
 	checkResults(t, dbPath)
 }
 
+func schemaOnlyTest(t *testing.T, dbName, filePrefix, sessionFile, dumpFilePath string) {
+	cmd := exec.Command("bash", "-c", fmt.Sprintf("go run github.com/cloudspannerecosystem/harbourbridge -driver mysqldump -schema-only -dbname %s -prefix %s < %s", dbName, filePrefix, dumpFilePath))
+	var out, stderr bytes.Buffer
+	cmd.Stdout = &out
+	cmd.Stderr = &stderr
+	if err := cmd.Run(); err != nil {
+		fmt.Printf("stdout: %q\n", out.String())
+		fmt.Printf("stderr: %q\n", stderr.String())
+		t.Fatal(err)
+	}
+	if _, err := os.Stat(fmt.Sprintf("%sreport.txt", filePrefix)); os.IsNotExist(err) {
+		t.Fatalf("report file not generated during schema-only test")
+	}
+	if _, err := os.Stat(fmt.Sprintf("%sschema.ddl.txt", filePrefix)); os.IsNotExist(err) {
+		t.Fatalf("legal ddl file not generated during schema-only test")
+	}
+	if _, err := os.Stat(fmt.Sprintf("%sschema.txt", filePrefix)); os.IsNotExist(err) {
+		t.Fatalf("readable schema file not generated during schema-only test")
+	}
+	if _, err := os.Stat(sessionFile); os.IsNotExist(err) {
+		t.Fatalf("session file not generated during schema-only test")
+	}
+}
+
+func dataOnlyTest(t *testing.T, dbName, dbURI, filePrefix, sessionFile, dumpFilePath string) {
+	cmd := exec.Command("bash", "-c", fmt.Sprintf("go run github.com/cloudspannerecosystem/harbourbridge -driver mysqldump -data-only -instance %s -dbname %s -prefix %s -session %s < %s", instanceID, dbName, filePrefix, sessionFile, dumpFilePath))
+	var out, stderr bytes.Buffer
+	cmd.Stdout = &out
+	cmd.Stderr = &stderr
+	cmd.Env = append(os.Environ(),
+		fmt.Sprintf("GCLOUD_PROJECT=%s", projectID),
+	)
+	if err := cmd.Run(); err != nil {
+		fmt.Printf("stdout: %q\n", out.String())
+		fmt.Printf("stderr: %q\n", stderr.String())
+		t.Fatal(err)
+	}
+	defer dropDatabase(t, dbURI)
+	checkResults(t, dbURI)
+}
+
+func TestMainApp(t *testing.T) {
```

Comment:
I made them inside a common test because data only uses the session file generated from the schema-only test. This ensures schema-only runs before data-only.

Reply:
I see that this is resolve now in the latest patch.